secrets/onepassword/main/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	op "github.com/ameier38/onepassword"
+)
+
+func main() {
```

Comment:
What is this command and relative Dockerfile used for?

Reply:
This was used for troubleshooting. I will remove it.

Reply:
Fixed in https://github.com/ContainerSolutions/externalsecret-operator/commit/a90f3d02e349a6ce8a9de027b35aa653f60cf369

Reply:
Did you forget to `git add`?. It looks like its still there :smile: